src/pages/posts/[postSlug]/index.js

```diff
@@ -22,7 +23,10 @@ export function PostComponent({ post }) {
       />
 
       <Main className="container">
-        <ContentWrapper content={post?.content()} />
+        <ContentWrapper content={post?.content()}>
+          <TaxonomyTerms post={post} taxonomy={'categories'} />
```

Comment:
You may want to add a key prop here to remove this warning
<img width="1791" alt="Screenshot 2022-03-22 at 09 45 12" src="https://user-images.githubusercontent.com/328805/159452915-1023014b-e5a7-4095-a868-0595d5a1cf51.png">